packages/ppbgdi/data_stream/python_logs/elasticsearch/ingest_pipeline/default.yml

```diff
@@ -0,0 +1,85 @@
+---
+description: Pipeline for processing sample logs
+processors:
+  - set:
+      field: ecs.version
+      value: '8.11.0'
+  - set:
+      field: event.ingested
+      value: '{{_ingest.timestamp}}'
+  - rename:
+      field: message
+      target_field: event.original
+      ignore_missing: true
+      if: 'ctx.event?.original == null'
+  - remove:
+      field: message
+      ignore_missing: true
+      if: 'ctx.event?.original != null'
+  - grok:
+      field: event.original
+      patterns:
+        - ^%{CHAR:first_char}
+      pattern_definitions:
+        CHAR: .
+#   - pipeline:
+#       if: ctx.first_char != '{'
+#       name: '{{ IngestPipeline "default-common" }}'
+  - pipeline:
+      if: ctx.first_char == '{'
+      name: '{{ IngestPipeline "default-json" }}'
+  - pipeline:
+      if: ctx.first_char == '{' && ctx?.service?.name != null && ctx?.service?.name == 'service-wmts'
+      name: '{{ IngestPipeline "wmts-json" }}'
+  - remove:
+      field:
+        - first_char
+  - set:
+      field: event.outcome
+      value: failure
+      if: 'ctx?.http?.response?.status_code != null && ctx?.http?.response?.status_code >= 400'
+  - set:
+      field: event.outcome
+      value: failure
+      if: "ctx?.event?.severity >= 40"
```

Comment:
- `status_code` should dominate `severity`
- failure for `status_code >= 500`

Reply:
Done